static/css/main.css

```diff
@@ -11,15 +11,27 @@ label { display:inline-block; font-weight: 600; margin-bottom: 0.25rem; width:20
 }
 
 .field-container {
-    display:block;
     margin: 0 0 0.2rem 0;
     position: relative;
 }
 
-.field-container span {
+.hidden-container {
+    display:none;
+}
+
+
+.field-container__span {
     position: relative;
 }
 
+.group-title{
+    display:none;
+}
+
+.metadata-fields{
+    display:none;
+}
+
 #schema-url-btn {
```

Comment:
This class should be renamed `.btn--small` and have every line bar the last 3 removed. Then this class can be added to the `schema-url-btn` in the html. It would remove a lot of duplication alongside removing the other classes I've suggested

Reply:
Updated now 👍